neurocurator/mainWin.py

```diff
@@ -1081,10 +1081,10 @@ def clearPaper(self):
     def waitForOCR(self, paperId, notify):
         
         while(not self.restClient.checkOCRFinished(paperId, self.dbPath)):
-            self.window.statusLabel.setText("Performing OCR...")
+            self.statusBar().showMessage("Performing OCR...")
             time.sleep(5)          
         
-        self.window.statusLabel.setText("OCR finished.")
+        self.statusBar().showMessage("OCR finished.")
```

Comment:
Should use argument `timeout` [doc](https://doc.qt.io/qt-5/qstatusbar.html#showMessage). I used 10 * 1000 elsewhere (Zotero database refresh).

Reply:
Well, if you want to refactor these things, sure. This PR was just about fixing the error with the use of the statusLabel widget. The rest is currently unchanged.